frontend/src/static/css/v_authorship.scss

```diff
@@ -2,6 +2,14 @@
 
 /* Authorship */
 #tab-authorship {
+  .toolbar {
+    > a {
```

Comment:
may I ask why this CSS change is needed?

Reply:
for `show all file details` and `hide all file details` to show up on top / below of each other.
I changed the condition for `show all file details` to `activeFilesCount > 0` from `activeFilesCount === 0`, which can be helpful since we now collapse files with > 2000 loc by default.

Reply:
ahhh okay, @damithc are u ok with the following, where if there's at least one file that is collapsed and at least one file that is showed, then the `show all file details` and `hide all file details` options will appear.

<img width="869" alt="Screen Shot 2020-02-20 at 11 23 46 PM" src="https://user-images.githubusercontent.com/32864116/74949018-18d90900-5438-11ea-968e-48456adee981.png">

Reply:
Yes I'm OK with it.